### ISSUE TITLE
Update fields for modulemd 1.0

### ISSUE THREAD

**Issue by ralphbean** [fedora-modularity/product-definition-center#8]
- version becomes stream.
- release becomes version.

Very confusing, but MBS and pdc-updater already have patches.  Once PDC is patched, it should return to being a bit simpler.

**Comment by ralphbean** [comment]
Before we can do this we have to solve fedora-modularity/product-definition-center#8

**Comment by nphilipp** [comment]
@ralphbean shoudn't this comment reference another issue and not this one?

**Comment by ralphbean** [comment]
Done, now.
